airflow/models/__init__.py

```diff
@@ -1856,6 +1851,13 @@ def init_run_context(self, raw=False):
         self.raw = raw
         self._set_context(self)
 
+    @staticmethod
+    @provide_session
+    def fetch_tis_db(dag_id, execution_date, session=None):
```

Comment:
Is this method used anywhere? Couldn't find any reference to it. If not, I'd suggest deleting it.

Reply:
👍